packages/core/saved-objects/core-saved-objects-migration-server-internal/src/next.ts

```diff
@@ -139,6 +139,7 @@ export const nextActionMap = (
       Actions.createIndex({
         client,
         indexName: state.tempIndex,
+        aliases: [state.tempIndexAlias],
```

Comment:
Is the alias automatically deleted when we associated index is removed (asking before the PR apparently doesn't add logic to delete the alias when the temp index is delete)?

Reply:
Yes, aliases are "attached" to an index.